app/build.gradle

```diff
@@ -469,6 +474,30 @@ android.applicationVariants.all { variant ->
     }
 }
 
+// -------------------------------------------------------------------------------------------------
+// Sentry: Read token from locale file if it exists (Only release builds)
+// -------------------------------------------------------------------------------------------------
+
+android.applicationVariants.all { variant ->
+    def variantName = variant.getName()
+
+    print("Sentry token: ")
+
+    if (variantName.contains("Release")) {
```

Comment:
We only want the Adjust token in release builds. However here I think we can be more open and allow the usage of sentry whenever there's a .sentry_token file present. It would be pretty nice to get crashes for our preview builds too.